test/functional/stats_controller_test.rb

```diff
@@ -8,8 +8,10 @@ class StatsControllerTest < ActionController::TestCase
       @number_of_downloads = 42
       rails_cinco = create(:rubygem, name: 'rails_cinco', number: 1)
 
-      Rubygem.stubs(:total_count).returns @number_of_gems
-      User.stubs(:count).returns @number_of_users
+      @rubygem_spy = Spy.on(RestClient, :post)
```

Comment:
Is this needed?

Reply:
It was not! (copy/paste error), will remove.